src/alarms/contacts/NewContact.js

```diff
@@ -31,55 +31,82 @@ class NewContact extends Component {
     return false;
   }
   handleClickCreateContactButton() {
-    this.setState({
-      submitActive: true
-    });
+    
     const { organisation, history } = this.props;
     const firstName = document.getElementById("firstName").value;
     const lastName = document.getElementById("lastName").value;
     const email = document.getElementById("email").value;
     const phoneNumber = document.getElementById("phoneNumber").value;
 
-    if (
-      firstName.length > 0 &&
-      lastName.length > 0 &&
-      this.validateEmail(email) &&
-      phoneNumber.length > 0
-    ) {
-      fetch("/api/v3/contacts/", {
-        credentials: "same-origin",
-        method: "POST",
-        headers: { "Content-Type": "application/json" },
-        body: JSON.stringify({
-          first_name: firstName,
-          last_name: lastName,
-          email: email,
-          phone_number: phoneNumber,
-          organisation: organisation.uuid
-        })
-      })
-        .then(response => {
-          if (response.status === 201) {
-            return response.json();
-          } else if (response.status === 403) {
-            alert("Not authorized");
-            return response.json();
-          } else {
-            alert("An error occurred while trying to create a new contact");
-            return response.json();
-          }
-        })
-        .then(data => {
-          history.push("/alarms/contacts");
-        });
-    } else {
-      alert("Please provide valid values in all fields!");
+    const firstNameTranslation = this.props.intl.formatMessage({ id: "contacts_app.first_name_3" });
+    const lastNameTranslation = this.props.intl.formatMessage({ id: "contacts_app.last_name_3" });
+    const emailTranslation = this.props.intl.formatMessage({ id: "contacts_app.email_address_3" });
+    // const phoneNumberTranslation = this.props.intl.formatMessage({ id: "contacts_app.phone_number_3" });
+    const fieldsDidnotValidateTranslation =  this.props.intl.formatMessage({ id: "contacts_app.following_fields_didnot_validate" }) + ' \n '; 
+
+    let errorMessage = fieldsDidnotValidateTranslation;
+    let showErrorMessage = false;
+
+    if ( firstName.length == 0 ) {
```

Comment:
Should we use === instead of ==? and when do you use == and ===? For me, I would always use === except for some cases where only the == sign would give me the required result.

Reply:
True. Will change this. It is better to be stricter.